src/cow-react/modules/swap/services/ethFlow/steps/signEthFlowOrderStep.ts

```diff
@@ -64,12 +62,13 @@ export async function signEthFlowOrderStep(
     value: orderParams.sellAmountBeforeFee.quotient.toString(),
   })
 
+  const orderDigest = hashOrder(getDomain(orderParams.chainId), order)
   // Generate the orderId from owner and validTo
   const orderId = packOrderUidParams({
-    // TODO: check orderDigest and validTo time
-    orderDigest: HashZero,
+    orderDigest,
     owner: ethFlowContract.address,
-    validTo: MAX_VALID_TO_EPOCH,
+    // TODO: check this, do we set MAX here or is that in contract?
```

Comment:
Yes, for identifying the order by its unique orderUid (such that later you can query it under the orders/UID API), you have to set the MAX value here.

Reply:
so userValidTo here is wrong yes?